package.json

```diff
@@ -64,11 +62,13 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "rimraf": "^5.0.5",
-    "storybook": "^7.5.3",
+    "storybook": "^8.0.2",
     "tinymce-4": "npm:tinymce@^4",
     "tinymce-5": "npm:tinymce@^5",
     "tinymce-6": "npm:tinymce@^6",
-    "typescript": "~5.2.2"
+    "tinymce-7": "npm:tinymce@^7",
+    "typescript": "~5.4.3",
+    "vite": "^5.2.2"
   },
   "version": "4.3.3-rc",
```

Comment:
Maybe it's your process to do this later but it seems these changes will need a major version bump.